### PR TITLE
translated docs/recipes/styling-css

### PR DESCRIPTION
**Pull request by k-guy-dev**
completed the translation of doc/recipes/styling-css

### CODE REVIEW

File: docs/docs/recipes/styling-css.md [hunk 1]
```diff
@@ -3,18 +3,18 @@ title: "Recipes: Styling with CSS"
 tableOfContentsDepth: 1
 ---
 
-There are so many ways to add styles to your website; Gatsby supports almost every possible option, through official and community plugins.
+आपल्या वेबसाइटवर शैली जोडण्याचे बरेच मार्ग आहेत; अधिकृत आणि समुदाय प्लगइनद्वारे गॅट्सबी जवळजवळ प्रत्येक संभाव्य पर्यायाचे समर्थन करते.
 
-## Using global CSS files without a Layout component
+## वापरत आहे global CSS फायली विना Layout component
```
Comment (spielers) [inline docs/docs/recipes/styling-css.md]: global CSS फायल विना वापरण्यासाठी

File: docs/docs/recipes/styling-css.md [hunk 2]
```diff
@@ -3,18 +3,18 @@ title: "Recipes: Styling with CSS"
...
 
-### Prerequisites
+### पूर्व शर्ती
 
-- An existing [Gatsby site](/docs/quick-start/) with an index page component
-- A `gatsby-browser.js` file
+- विद्यमान [Gatsby site](/docs/quick-start/) सह index page component
```
Comment (spielers) [inline docs/docs/recipes/styling-css.md]: retranslate this, please.

File: docs/docs/recipes/styling-css.md [hunk 3]
```diff
@@ -3,18 +3,18 @@ title: "Recipes: Styling with CSS"
...
+- एक `gatsby-browser.js` फाईल
 
-### Directions
+### दिशानिर्देश
 
-1. Create a global CSS file as `src/styles/global.css` and paste the following into the file:
+1. तयार करा एक global CSS फाईल म्हणून `src/styles/global.css` आणि खालील फाईलमध्ये पेस्ट करा:
```
Comment (spielers) [inline docs/docs/recipes/styling-css.md]: global CSS फाईल म्हणून `src/styles/global.css` तयार करा आणि खालील फाईलमध्ये पेस्ट करा :

**Review by spielers — CHANGES_REQUESTED** [review]
Hey @k-guy-dev  please retranslate the entire document once again with Marathi sentence Grammer keeping in mind.

And also Let us know if you are aiming for hacktoberfest.

Best,
spielers

**Comment by k-guy-dev** [comment]
> Hey @k-guy-dev please retranslate the entire document once again with Marathi sentence Grammer keeping in mind.
> 
> And also Let us know if you are aiming for hacktoberfest.
> 
> Best,
> spielers

yup @spielers ,, I'm aiming for hacktoberfest

File: docs/docs/recipes/styling-css.md [hunk 4]
```diff
@@ -3,18 +3,18 @@ title: "Recipes: Styling with CSS"
 tableOfContentsDepth: 1
 ---
 
-There are so many ways to add styles to your website; Gatsby supports almost every possible option, through official and community plugins.
+आपल्या वेबसाइटवर शैली जोडण्याचे बरेच मार्ग आहेत; अधिकृत आणि समुदाय प्लगइनद्वारे गॅट्सबी जवळजवळ प्रत्येक संभाव्य पर्यायाचे समर्थन करते.
 
-## Using global CSS files without a Layout component
+## global CSS फायल विना वापरण्यासाठी
```
Comment (visshaljagtap) [inline docs/docs/recipes/styling-css.md]: लेआउट कौम्पोनॅन्ट विना global CSS फाईल्स वापरणे

File: docs/docs/recipes/styling-css.md [hunk 5]
```diff
@@ -3,18 +3,18 @@ title: "Recipes: Styling with CSS"
...
 
-### Prerequisites
+### पूर्व शर्ती
 
-- An existing [Gatsby site](/docs/quick-start/) with an index page component
-- A `gatsby-browser.js` file
+- [Gatsby site](/docs/quick-start/) index page component
```
Comment (visshaljagtap) [inline docs/docs/recipes/styling-css.md]: [Gatsby site](/docs/quick-start/) index page component -> विद्यमान [Gatsby site](/docs/quick-start/) अनुक्रमणिका page कौम्पोनॅन्ट

**Review by visshaljagtap — CHANGES_REQUESTED** [review]

**Comment by spielers** [comment]
Hey @k-guy-dev Thanks for your contributions but I would suggest you should translate something from the `tutorials/` section

**Review by spielers — APPROVED** [review]
Thanks @k-guy-dev

**Comment by spielers** [comment]
Thanks, @k-guy-dev  💐

**Pull request merged**